atlasdb-client/src/main/java/com/palantir/atlasdb/stream/PersistentStreamStore.java

```diff
@@ -68,5 +72,5 @@
      * because of this complexity assuming the stream isn't too large (100MB or less).
      */
     Pair<Long, Sha256Hash> storeStream(InputStream stream);
-
+    Map<Long, Sha256Hash> storeStreams(Transaction t, Map<Long, InputStream> streams);
```

Comment:
I made this take a transaction to help with correctness even if the user does not pass an reentrant transaction.  I can remove it if it is worth it to make the api cleaner.